app/models/warehouse/case_report.rb

```diff
@@ -281,10 +281,15 @@ def process_class_related_process(kase, case_report)
         end
       end
 
+      def total_cached_num_pages(kase)
+        kase.data_request_areas&.decorate&.sum(&:cached_num_pages) || 0
+      end
+
       def process_offender_sar(kase, case_report)
         case_report.third_party_company_name = kase.third_party_company_name
         case_report.number_of_exempt_pages = kase.number_exempt_pages
         case_report.number_of_final_pages = kase.number_final_pages
+        case_report.cached_num_pages = total_cached_num_pages(kase)
```

Comment:
"cached_num_pages" doesn't feel like the best name for this attribute. Maybe "pages_received" instead?